tests/integration_tests/resources/dsl/agent_tests/plugin-requires-old-package-blueprint/plugins/requires-old-package-plugin/setup.py

```diff
@@ -22,6 +22,6 @@
     packages=['requires_old_package_plugin'],
     install_requires=[
         # The old repo/package before we merged it to cloudify-common
-        "cloudify-plugins-common>=5.1.0.dev1"
+        "cloudify-common==5.0.0"
```

Comment:
Why downgraded?

Reply:
Well, for one, cloudify-plugins-common has no version above 4.3.2.
cloudify-common, which we've been using since, has of course 5.1.0.dev1, but the test is meant to check whether a plugin which requires an *older* package can work. 5.0.0 is the earliest package which doesn't error for reasons such as changing how our MQ works.